rust/headless-client/src/lib.rs

```diff
@@ -207,9 +215,9 @@ fn run_standalone(cli: Cli, rt: tokio::runtime::Runtime, token: &SecretString) -
         rt.handle().clone(),
     );
     // TODO: this should be added dynamically
```

Comment:
So right now roaming will break DNS?

Reply:
I made a TODO to test that, it should still work for the GUI since the GUI will listen for network changes and notify connlib. But the headless / standalone Client doesn't do that yet.

Reply:
I checked it out, the GUI does still notify the IPC service / connlib about network up/down and DNS changes.
But in doing so I also saw that we get `on_update_resources` about once per second after I change networks. That doesn't seem right, I'll try to replicate it on main and open an issue.